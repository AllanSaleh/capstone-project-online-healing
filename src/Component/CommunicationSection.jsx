import React from 'react';

const CommunicationSection = () => {
    return (  );
}
 
export default CommunicationSection;
