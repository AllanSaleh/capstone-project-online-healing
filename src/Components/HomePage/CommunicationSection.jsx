import React from 'react';

import Card from './Card';

import PhoneCell from './Images/phone-cell.svg';
import ChatIcon from './Images/chat-icon.svg';
import VideoCallIcon from './Images/video-call-icon.svg';

const CommunicationSection = () => (
  <div className=" md:h-section flex flex-col justify-evenly px-sides">
    <h1 className="sm:text-xl md:text-4xl xl:text-title pb-2 uppercase mb-2">
      we can communicate through
    </h1>
    <div className="flex flex-col md:flex-row items-center md:items-start md:justify-around">
      <Card
        showIcon
        icon={PhoneCell}
        title="voice call"
        content="Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!"
        showButton={false}
      />

      <Card
        showIcon
        icon={ChatIcon}
        title="chat"
        content="Need to talk to someone?Come have a chat with your therapist!"
        showButton={false}
      />

      <Card
        showIcon
        icon={VideoCallIcon}
        title="video call"
        content="For better experience therapists recommend video calls, but always remember that its a choice!"
        showButton={false}
      />
    </div>
  </div>
);

export default CommunicationSection;
