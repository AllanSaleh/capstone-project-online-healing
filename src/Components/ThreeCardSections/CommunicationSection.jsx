import React from 'react';
import Card from './Card';
import PhoneCell from '../../Icons/phone-cell.svg';
import ChatIcon from '../../Icons/chat-icon.svg';
import VideoCallIcon from '../../Icons/video-call-icon.svg';

const CommunicationSection = () => (
  <div className="h-section flex flex-col justify-evenly px-sides">
    <h1 className="text-title pb-2 uppercase mb-2">we can communicate through</h1>
    <div className="flex justify-around">
      <Card
        showIcon
        icon={PhoneCell}
        title="voice call"
        content="Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!"
        showButton={false}
      />

      <Card
        showIcon
        icon={ChatIcon}
        title="chat"
        content="Need to talk to someone?Come have a chat with your therapist!"
        showButton={false}
      />

      <Card
        showIcon
        icon={VideoCallIcon}
        title="video call"
        content="For better experience therapists recommend video calls, but always remember that its a choice!"
        showButton={false}
      />
    </div>
  </div>
);

export default CommunicationSection;
