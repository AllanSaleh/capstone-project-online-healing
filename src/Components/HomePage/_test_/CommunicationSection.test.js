import * as React from 'react';
import * as renderer from 'react-test-renderer';
import CommunicationSection from '../CommunicationSection';

it('Communication Section renders correctly', () => {
  const tree = renderer.create(<CommunicationSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
