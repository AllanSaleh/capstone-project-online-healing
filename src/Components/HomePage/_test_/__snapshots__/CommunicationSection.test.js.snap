// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Communication Section renders correctly 1`] = `
<div
  className=" md:h-section flex flex-col justify-evenly px-sides"
>
  <h1
    className="sm:text-xl md:text-4xl xl:text-title pb-2 uppercase mb-2"
  >
    we can communicate through
  </h1>
  <div
    className="flex flex-col md:flex-row items-center md:items-start md:justify-around"
  >
    <div
      className="h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg rounded-lg md:w-1/4 p-4"
    >
      <img
        alt=""
        className="flex-initial mb-4"
        src="phone-cell.svg"
      />
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        voice call
         Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
        $
      </p>
    </div>
    <div
      className="h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg rounded-lg md:w-1/4 p-4"
    >
      <img
        alt=""
        className="flex-initial mb-4"
        src="chat-icon.svg"
      />
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        chat
         Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        Need to talk to someone?Come have a chat with your therapist!
        $
      </p>
    </div>
    <div
      className="h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg rounded-lg md:w-1/4 p-4"
    >
      <img
        alt=""
        className="flex-initial mb-4"
        src="video-call-icon.svg"
      />
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        video call
         Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        For better experience therapists recommend video calls, but always remember that its a choice!
        $
      </p>
    </div>
  </div>
</div>
`;
